llvm/test/tools/yaml2obj/GOFF/GOFF-basic.yaml

```diff
@@ -0,0 +1,59 @@
+# RUN: yaml2obj %s | od -v -An -tx1 | FileCheck --ignore-case %s
```

Comment:
This test is in a `GOFF` subdirectory. Tere's no need to prefix the filename  with `GOFF-`.

What is this test actually trying to test that isn't tested in other tests? Please add a comment to explain the test's purpose.

Same comments apply to the other tests.